### PR TITLE
Added previously missing field-descriptors

### PR DESCRIPTION
**Pull request by turbo-ele**

### CODE REVIEW

File: src/Sulu/Bundle/ContactBundle/Controller/AccountController.php [hunk 1]
```diff
@@ -1235,6 +1235,19 @@ protected function initFieldDescriptors()
     {
         $this->fieldDescriptors = [];
 
+        $accountAddressJoin = [
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContactBundle/Controller/AccountController.php]: what is when there are more than one address?

Comment (turbo-ele) [inline src/Sulu/Bundle/ContactBundle/Controller/AccountController.php]: there is just one main address

Comment (wachterjohannes) [inline src/Sulu/Bundle/ContactBundle/Controller/AccountController.php]: ok i will try this after lunch

File: src/Sulu/Bundle/ContactBundle/Controller/AccountController.php [hunk 2]
```diff
@@ -1294,23 +1307,53 @@ protected function initFieldDescriptors()
...
+            'zip',
+            self::$addressEntityName,
+            'contact.address.zip',
+            $accountAddressJoin,
+            false,
+            false,
+            'string'
+        );
+
+        $this->fieldDescriptors['state'] = new DoctrineFieldDescriptor(
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContactBundle/Controller/AccountController.php]: should not be default selected and also be able to hide (in list)

File: src/Sulu/Bundle/ContactBundle/Controller/AccountController.php [hunk 3]
```diff
@@ -1294,23 +1307,53 @@ protected function initFieldDescriptors()
...
-                    self::$accountAddressEntityName . '.address'
-                ),
-            ],
+            $accountAddressJoin,
             false,
             true,
             'string'
         );
 
+        $this->fieldDescriptors['zip'] = new DoctrineFieldDescriptor(
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContactBundle/Controller/AccountController.php]: can not be selected in column options

Comment (wachterjohannes) [inline src/Sulu/Bundle/ContactBundle/Controller/AccountController.php]: same as state (default not selected)

File: src/Sulu/Bundle/ContactBundle/Controller/AccountController.php [hunk 4]
```diff
@@ -1294,23 +1307,53 @@ protected function initFieldDescriptors()
...
+            'state',
+            self::$addressEntityName,
+            'contact.address.state',
+            $accountAddressJoin,
+            false,
+            false,
+            'string'
+        );
+
+        $this->fieldDescriptors['countryCode'] = new DoctrineFieldDescriptor(
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContactBundle/Controller/AccountController.php]: same as state

**Pull request merged**